x-pack/platform/plugins/shared/streams/server/routes/streams/processing/simulation_handler.ts

```diff
@@ -609,6 +637,11 @@ const isSuccessfulProcessor = (
 ): processor is WithRequired<IngestPipelineSimulation, 'doc' | 'tag'> =>
   processor.status === 'success' && !!processor.tag;
 
+const isSkippedProcessor = (
+  processor: IngestPipelineSimulation
+  // @ts-expect-error Looks like the IngestPipelineSimulation.status is not typed correctly and misses the 'skipped' status
```

Comment:
Looks like it's using `WatcherActionStatusOptions` which seems very wrong... Could you report this?

Reply:
Yep this is wrong, I'll report that! 👌

Reply:
https://github.com/elastic/elasticsearch-js/issues/2647